### ISSUE TITLE
MariaDB will not upgrade past 10.7.5

### ISSUE THREAD

**Issue by roncli** [roncli/roncli.com#445]
MariaDB is having a problem upgrading its Docker container to any version past 10.7.5.

Trying from 10.7.1 to 10.9.2:
```
[ERROR] InnoDB: Upgrade after a crash is not supported. The redo log was created with MariaDB 10.7.1.
```

Trying from 10.7.1 to 10.7.5 worked.

Trying from 10.7.5 to 10.8.2:
```
[ERROR] InnoDB: Upgrade after a crash is not supported. The redo log was created with MariaDB 10.7.1.
```

**Comment by roncli** [comment]
See MariaDb/mariadb-docker#456 for the upstream bug report.

**Comment by roncli** [comment]
Fixed by 337848316b8d96b96617616124eef4cba079d7e2.
